mods/gennext/statuses.js

```diff
@@ -22,7 +22,7 @@ exports.BattleStatuses = {
 			}
 		},
 		onEnd: function (target) {
-			this.battle.add('-curestatus', target, 'frz');
```

Comment:
...wait, wtf, does `battle.battle` exist? It probably shouldn't. Did this code not work before?

Reply:
No, it doesn't; this code was probably broken, but it only runs if someone gets unfrozen while playing Gen-NEXT, so it's not something that's going to get a lot of testing...